actors/builtin/miner/deadline_state.go

```diff
@@ -1261,6 +1281,9 @@ func (d *Deadline) IsLive() (bool, error) {
 		return true, nil
 	}
 
+	// TODO: Should the deadline be considered live if the sectors snapshot has changed?
```

Comment:
No there's nothing worth checking in the sectors snapshots.  Partition and post snapshots cover edge cases where deadline is considered live without having live sectors.  The sectors snapshot is more information taken from miner state than information about this deadline.

Reply:
Another way to put it is that any changes to sectors in this deadline require that those sectors are live which is covered by the first check in this method.